ADQL.tex

```diff
@@ -1409,6 +1417,33 @@ \subsubsection{Overview}
     \item REGION
 \end{itemize}
 
+\subsubsection{Language feature}
+\label{sec:functions.geom.feature}
+
+All functions described in this section use the following IVOID:
+
+\begin{verbatim}
+    ivo://ivoa.net/std/tapregext#features-adqlgeo
+\end{verbatim}
+    
+Each geometrical function is declared using this IVOID and its name.
+See \SectionRef{sec:capabilities} for more details.
+
+All other optional features have an IVOID starting with:
+
+\begin{verbatim}
+    ivo://ivoa.net/std/tapregext#features-adql-
+\end{verbatim}
+
+Note the ending hyphen. The IVOID for the geometry features does not have this
+hyphen. This is done on purpose. It is actually due to a typo when preparing the
+ADQL-2.1. This typo lasted until the first ADQL-2.1 implementations appeared in
```

Comment:
Ah... no, the reason actually is more serious: features-adqlgeo is already defined in TAPRegExt 1.0.  So, I guess we should just say "it's legacy, and the name was coined before it became clear that we will have a lot more features-adql-* identifiers" or so.

Reply:
> Ah... no, the reason actually is more serious: features-adqlgeo is already defined in TAPRegExt 1.0. So, I guess we should just say "it's legacy, and the name was coined before it became clear that we will have a lot more features-adql-* identifiers" or so.

Ho, I see. Is it better now?